README.md

```diff
@@ -1,10 +1,9 @@
 <a name="readme-top"></a>
 
 <div align="center">
+  <img src="budget-trans.png" alt="logo" width="240"  height="auto" />
   <br/>
-
   <h3><b>My Budget App</b></h3>
-
 </div>
 
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly your project works perfectly on my side but I can not use your `Live Demo` version, below I paste problem's `video`, please solve this:

![mobile](https://github.com/MMGGYY66/my-budget-app/assets/63915024/f2a217ce-e1bf-4005-999a-3fe7a79af514)

Reply:
done with new deployment : https://mohamed-budget-app.onrender.com/splash_screen